src/main/java/uk/gov/digital/ho/hocs/audit/core/RestHelper.java

```diff
@@ -42,27 +38,17 @@ private static String getBasicAuth(String basicAuth) {
     }
 
     public <R> R get(String rootUri, String endpoint, Class<R> type) {
-        try {
-            log.debug("Making GET request {}{}", rootUri, endpoint);
-            return restTemplate
-                    .exchange(rootUri.concat(endpoint), HttpMethod.GET, getAuthenticatedEntity(), type)
-                    .getBody();
-        } catch (Exception e) {
```

Comment:
These are actually handled in RestResponseEntityExceptionHandler->HttpClientErrorException